src/components/screens/bookmarks/addBookmark/addBookmark.js

```diff
@@ -99,7 +99,7 @@ const AddBookmark = ({
       {
         value: usernameValue,
         feedback: usernameFeedback,
-        readonly: isDelegate === true || usernameFeedback !== '',
+        readonly: isDelegate === 'true',
```

Comment:
should it be the string `'true'` or a boolean `true`?

Reply:
It comes from the query parameters. It's a string.